config/install/entity_browser.browser.tide_media_browser_iframe.yml

```diff
@@ -69,3 +69,14 @@ widgets:
     weight: -9
     label: 'Add audio'
     id: entity_form
+  514a147d-5e8c-4006-8bdd-6959b6239548:
```

Comment:
No need to add this here as this shouldn't be added if the module is enabled. If you put it in the tide_media_secure_files install hook, then every time it gets enabled, it will add this in this config.